include/ultra64/ucode.h

```diff
@@ -0,0 +1,27 @@
+#ifndef ULTRA64_UCODE_H
+#define ULTRA64_UCODE_H
+
+#include "types.h"
+
+#define SP_DRAM_STACK_SIZE8     (0x400)
+#define SP_DRAM_STACK_SIZE64    (SP_DRAM_STACK_SIZE8 >> 3)
```

Comment:
Is there a particular reason to use SIZE8/SIZE64 macros instead of handling it the same as `OS_YIELD_DATA_SIZE` with `/ sizeof(u64)` for example?

Reply:
The only reason is that's how they are in the original libultra headers

Reply:
I would personally go with `/ sizeof(u64)` since that fits the codebase a lot better, but I don't have a strong opinion on it

Reply:
Since there's no `SP_DRAM_STACK_SIZE` macro I'd either have to use `SP_DRAM_STACK_SIZE8`, or make a new macro to use `/ sizeof(u64)`. I'm not sure if it's worth it over just using `SP_DRAM_STACK_SIZE64`

Reply:
I was thinking `SP_DRAM_STACK_SIZE8` could simply be `SP_DRAM_STACK_SIZE` and wherever the u64 size is needed we would have `SP_DRAM_STACK_SIZE / sizeof(u64)`, but I might be missing something

Reply:
That would break compatibility with the original header, which so far (afaik) we've managed not to do anywhere (other than having incomplete/jumbled libultra headers in general), and I think it would be good to strive to maintain such compatibility.
That is, adding our own stuff or making equivalent replacements (like the `unsigned` to `u32` in the handwritten asm PR) would be fine, but renaming or entirely removing something wouldn't be